### DISCUSSION TITLE
Allow authenticated user to add row only with their userid in field?

### DISCUSSION THREAD

**Discussion by oguruma1218** [supabase/supabase#14191] [Questions] [answered]
I have a table called Collections and a Users table

Collections has a owner_user field which is related to Users (IDs). 

I want to let users add a row to Collections, but only let them insert their own User ID (so they can't create a Collection that belongs to somebody else). 

Is this possible with RLS?

**Comment by GaryAustin1** [comment, marked as answer]
You set default id to auth.uid and also on the insert policy check the column is auth.uid() so they can’t change it.
